src/Extensions/AbstractWebDriverExtension.php

```diff
@@ -17,33 +17,42 @@
 use Codeception\Exception\ModuleRequireException;
 use Codeception\Module\WebDriver;
 
+/**
+ * Base class for extensions
+ */
 abstract class AbstractWebDriverExtension extends AbstractCodeceptionExtension
 {
     /**
      * @var WebDriver
      */
-    protected $webDriverModule;
+    protected \Codeception\Module\WebDriver $webDriverModule;
 
     /**
      * @var string
      */
-    protected $environment;
+    protected string $environment = 'testing';
```

Comment:
Can you remove the superfluous @param and @return in the docblock they are already specified in the code. Thanks

Reply:
Using "testing" as a default is opinionated. E.g. all Symfony apps would by default have a test environment called "test".